src/test/resources/data/InstructorFeedbackResultsPageUiTest.json

```diff
@@ -1771,7 +1771,22 @@
       "recipient": "CFResultsUiT.benny.no@gmail.tmt",
       "responseMetaData": {
         "value": "Benny self feedback."
-      }
+      },
+      "giverSection": "None",
+      "recipientSection": "None"
+    },
+    "Response to Instructors": {
+      "feedbackSessionName": "Second Session",
+      "courseId": "CFResultsUiT.CS2104",
+      "feedbackQuestionId": "1",
+      "feedbackQuestionType": "TEXT",
+      "giver": "CFResultsUiT.alice.b@gmail.tmt",
```

Comment:
Is the giver a instructor?

Reply:
No. Giver is student. This response is to test for response to instructors for `RGQ` questions.  There are already `Instructor->Student` responses in the `json` file.

Reply:
I am thinking whether this testing data will confuse other developers as currently we dosen't support giving feedback to instructor from student side.